auto_landing/CMakeLists.txt

```diff
@@ -11,8 +11,14 @@ set(CMAKE_BUILD_TYPE Release)
 #cs_add_library(auto_landing_lib src/template_ros.cpp src/template_ros_pub.cpp)
 #target_link_libraries(auto_landing_lib ${PCL_LIBRARIES})
 
+cs_add_library(landing_lib src/ladnding.cpp)
```

Comment:
Spelling of landing is wrong, please correct. Also instead of writing landing_lib write ${PROJECT_NAME}_lib.